src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1447,7 +1453,7 @@ void runtime::cpu::CPU_ExternalFunction::build()
                         }
                     });
 
-                if (m_release_function)
+                if (m_release_function && !(std::getenv("NGRAPH_DEX_DEBUG") != nullptr))
```

Comment:
Double negations are harder to read: `!(std::getenv("NGRAPH_DEX_DEBUG") != nullptr)`
why not  `std::getenv("NGRAPH_DEX_DEBUG")  == nullptr` or `!std::getenv("NGRAPH_DEX_DEBUG")`

Reply:
Why is release_function impacted?